AppCenter/AppCenter/MSService.h

```diff
@@ -6,20 +6,21 @@
 @protocol MSService <NSObject>
 
 /**
- *  Enable/disable this service.
+ * Enable/disable this service.
+ * The state is persisted in the device's storage across application launches.
  *
- *  @param isEnabled whether this service is enabled or not.
+ * @param isEnabled whether this service is enabled or not.
```

Comment:
As we're touching this comment, how about

* capitalize "Whether"
* capitalize "A" in "A boolean..."

Actually, thinking of it, why not rewrite both comments for `isEnabled` into a full sentence? How about "A BOOL that indicates if this service is enabled or not"?